app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -57,4 +57,12 @@ ManageIQ.angular.app.controller('cloudNetworkFormController', ['$http', '$scope'
     $scope.angularForm.$setPristine(true);
     miqService.miqFlash("warn", "All changes have been reset");
   };
+
+  $scope.filterNetworkManagerChanged = function(id) {
+    miqService.sparkleOn();
+    $http.get('/cloud_network/cloud_tenants_by_ems/' + id).success(function(data) {
+      $scope.available_tenants = data.available_tenants;
+    });
```

Comment:
This pattern was remodeled recently into `$http.get().then().catch()` in our code.

Please look into `cloudSubnetFormController` for an example and change this for this to be consistent
with the rest.

Thanks.